### PR TITLE
Change of member site - change AVL membership level and add Autoware Foundation #105 # 106

### PR DESCRIPTION
**Pull request by ariexi**
Signed-off-by: ariexi <andreas.riexinger@de.bosch.com>

### CODE REVIEW

File: content/members/autoware-foundation.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: "The Autoware Foundation"
+date: 2020-09-15T11:54:37+03:00
+draft: false
+logo: "/images/members/autoware-foundation.png"
+website: "https://www.autoware.org/"
+participation_levels: "Guest"
+member_id: xxxx
```
Comment (autumnfound) [inline content/members/autoware-foundation.md:8]: If unknown, this field should be excluded. With it in, the URL for this member turns into a 404 (https://www.eclipse.org/membership/showMember.php?member_id=xxxx). In this case, the member ID should be 1462

**Review by autumnfound — CHANGES_REQUESTED** [review]
Additional to my other comment, the logos of members should be shrunk/compressed to make the page responsive and to be more easily accessible. We typically aim for 25KB, with a maximum of ~50KB

File: content/members/autoware-foundation.md (hunk 1 above)
Comment (ariexi) [reply content/members/autoware-foundation.md]: @autumnfound Ok, thanks, got it!.
Now I changed it. ;-)

Comment (ariexi) [reply content/members/autoware-foundation.md]: @autumnfound Ok, thanks, got it!.
Now I changed it. ;-)

**Review by autumnfound — APPROVED** [review]

**Pull request merged**